src/components/LogPane/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { AutoSizer, List, ListProps } from "react-virtualized";
+
+type LogPaneProps = Omit<
+  ListProps,
+  "height" | "width" | "itemData" | "rowHeight"
+> & {
+  logLines: string[];
+  wrap: boolean;
+};
+
+const LogPane: React.FC<LogPaneProps> = ({
```

Comment:
Very mysterious but I think the last log line isn't getting rendered properly 🤔


https://user-images.githubusercontent.com/47064971/188020129-697c7aa0-2dbc-4fc0-8d12-6d599c097900.mov

I think there should be a line saying `[j0:sec1] Starting mongod on port 20002...` at the end